services/datamanager/datasync/sync.go

```diff
@@ -149,6 +163,17 @@ func (s *syncer) syncDataCaptureFile(f *datacapture.File) {
 		if err != nil {
 			s.syncErrs <- errors.Wrap(err, "error closing data capture file")
 		}
+
+		errStatus := status.Convert(uploadErr)
```

Comment:
Since this block:
```
s := status.Convert(err)
if s.Code() == codes.InvalidArgument {
	...
}
```
exists in three palces now (here, syncarbitraryfile, and epxonentialRetry), I think it would make sense to pull into some `isRetryableGRPCError(err) bool` function